src/wayland/libgwater-wayland.c

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
As far as I can tell from the Makefile, we generate this code. Is there an apparent reason to check it in into the repo?

Reply:
No it's taken from libgwater, see https://github.com/dunst-project/dunst/pull/781#issuecomment-731775356.
The files in `src/wayland/protocols/` are generated, but since not all distros have a recent version fo `wayland-scanner` (which generates these files) or `wayland-protocols`. That's why these files are checked in the repo and are not generated on compile time (yet).